util/topgen/lib.py

```diff
@@ -365,3 +406,15 @@ def get_base_and_size(name_to_block: Dict[str, IpBlock],
         assert isinstance(base_addr, int)
 
     return (base_addr, size_byte)
+
+
+def get_io_enum_literal(sig, prefix):
+    """Returns the DIO pin enum literal with value assignment"""
+    if sig['idx'] < 0:
+        return_str = Name.from_snake_case("{}_{}".format(
```

Comment:
* Why combine it first and then have from_snake_case split it up again?
* Can you rename `return_str` or return directly? return_str isn't a string, but a Name.

Reply:
Yeah that seems unnecessary.
I've also change this to remove the camel case variant directly instead of assigning to a temporary var.